water-sa.csl

```diff
@@ -71,15 +71,15 @@
       </name>
     </names>
   </macro>
-  <macro name="interviewer">
+  <!--<macro name="interviewer">
     <text term="interviewer" form="verb" text-case="capitalize-first" suffix=" "/>
     <names variable="interviewer">
       <name and="text" delimiter=", " delimiter-precedes-last="never">
         <name-part name="family" text-case="uppercase"/>
         <name-part name="given"/>
       </name>
     </names>
-  </macro>
+  </macro>-->
```

Comment:
@libora, in general, please just delete unused macros instead of commenting them out.